src/helpers/getPrettierInstance.js

```diff
@@ -4,11 +4,13 @@ const bundledPrettier = require('prettier');
 const { getCurrentFilePath } = require('../editorInterface');
 const { findCachedFromFilePath } = require('./general');
 const path = require('path');
+const { findCached } = require('atom-linter');
+const globalModules = require('global-modules');
```

Comment:
Does this work for globally installed packages in Yarn? The repo makes it sound like it only works for npm? We should really support both if possible.